digital-deck-server/services/utilities.js

```diff
@@ -1,23 +1,38 @@
 const Session = require('../models/session');
+const sessionData = require('../models/dbConnection');
 
-let concSessions = [];
+
+/**
+ * Cleans sessions from the database
+ * 
+ * @return [array of {json} ] The array of json containing information of all sessions
+ */
+function cleanSession(session) {
+    delete session._id;
+}
 
 /**
  * Returns all the sessions
  * 
  * @return [array of {json} ] The array of json containing information of all sessions
  */
-function getConcSessions()
+async function getConcSessions()
 {
-    return concSessions;
+    var sessions = await sessionData.find();
+    console.log(sessions);
+    sessions.forEach((session) => {
+        session = cleanSession(session);
```

Comment:
Why do we have to do this?

Reply:
When we store session to database it creates it's own _id field which I am removing here before sending it to user

Reply:
Ohh I see. So the client should get the correct sessionId in the DB yeah?

Reply:
That and it was unnecessary info for client side. We don't need to remove it but this just keeps the session object identical as before to ensure nothing breaks on UI